pennylane/transforms/zx/todd.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018-2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains a transform to apply the
+`phase_block_optimize <https://pyzx.readthedocs.io/en/latest/api.html#pyzx.optimize.phase_block_optimize>`__
+pass (available through the external `pyzx <https://pyzx.readthedocs.io/en/latest/index.html>`__ package)
+to a PennyLane Clifford + T circuit.
+"""
+
+from pennylane.tape import QuantumScript, QuantumScriptBatch
+from pennylane.transforms import transform
+from pennylane.typing import PostprocessingFn
+
+from .converter import from_zx, to_zx
+from .helper import _needs_pyzx
+
+
+@_needs_pyzx
+@transform
+def todd(tape: QuantumScript) -> tuple[QuantumScriptBatch, PostprocessingFn]:
+    """
+    Apply the `Third Order Duplicate and Destroy (TODD) <https://arxiv.org/abs/1712.01557>` algorithm to reduce
```

Comment:
```suggestion
    Apply the `Third Order Duplicate and Destroy (TODD) <https://arxiv.org/abs/1712.01557>`__ algorithm to reduce
```
something something Sphinx anonymous links *grumble grumble*

Reply:
Ah, actually:
- Without any `_`, we get italics instead of a link!
- With one `_`, the link can be referenced (rarely needed)
- With two `__`, the link can not be referenced (its "anonymous") but Sphinx will also not complain to you if you include the same link multiple times (which it will if you use just one `_`).

So basically just use double `_` unless there is a concrete reason not to :D